CMakeLists.txt

```diff
@@ -479,6 +512,12 @@ add_subdirectory(core)
 # Note that rocprofiler-register is in base and is what higher level clients
 # depend on. The profiler itself is independent.
 add_subdirectory(profiler)
+# Data center tools (RDC, etc.)
+if(THEROCK_ENABLE_DC_TOOLS AND THEROCK_ENABLE_RDC AND NOT WIN32)
+  add_subdirectory(dctools)
+elseif(THEROCK_ENABLE_RDC)
+  message(WARNING "THEROCK_ENABLE_RDC is ON but RDC requires Linux and THEROCK_ENABLE_DC_TOOLS=ON. Skipping.")
```

Comment:
Fail fast / FATAL_ERROR please. Warnings for illegal things get buried in CI logs and regress silently.